tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -273,6 +273,8 @@ copy_kernel_modules()
 	info "Copy kernel modules from ${KERNEL_MODULES_DIR}"
 	mkdir -p "${dest_dir}"
 	cp -a "${KERNEL_MODULES_DIR}" "${dest_dir}/"
+	local KERNEL_VER=$(ls ${dest_dir})
```

Comment:
Any chance of having modules for more than one kernel on $dest_dir? If yes, than you should probably pick one or depmod can handle more than one version at once?

Reply:
If this directory is the output of the build_image script, then there will be only one. But this could be used more generically if it is ran outside of the build_image script and it would be possible to have multiple directories, which would make a loop necesary.